src/components/Modals/CartModal.js

```diff
@@ -0,0 +1,33 @@
+import {
+  BackGround,
+  ModalContainer,
+  ConfirmRow,
+  ClearAll,
+  RemainAll,
+  ModalTitle,
+  ModalBody,
+} from "./styled";
+
+const CartModal = ({ showModal, deleteAll, noChange }) => {
+  return (
+    <>
+      {showModal ? (
```

Comment:
Just FYI, another way to implement it is:
if(!showModal){ return null; }
return <BackGround>....